angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.stories.ts

```diff
@@ -268,6 +268,26 @@ export const defaultStory = () => ({
           Sed ut perspiciatis unde omnis iste natus error sit <a href="#nogo" sprkLink class="sprk-b-Link--inline-light"> inline link</a> accusantium doloremque laudantium
         </p>
       </sprk-stack>
+      <div appStoreLinksSlot>
+        <div sprkStackItem>
+          <sprk-stack
+            itemSpacing='medium'
+            splitAt='tiny'
```

Comment:
Looks like there are double quotes and single quotes here. We use double quotes for props. Lets make the single quote prop values into double.

Reply:
stack stories have single quotes apparently in the copy code. fixed this and those stories!